assets/js/blocks/cart-checkout/mini-cart/block.tsx

```diff
@@ -113,7 +114,19 @@ const MiniCartBlock = ( {
 		};
 	}, [] );
 
-	const subTotal = getSetting( 'displayCartPricesIncludingTax', false )
+	const showIncludingTax = getSettingWithCoercion(
+		'displayCartPricesIncludingTax',
+		false,
+		isBoolean
+	);
+
+	const labelTaxName = getSettingWithCoercion(
+		'labelIncludingTax',
```

Comment:
I feel this setting name might be confusing, given that the value might be _(**excl.** VAT)_. What about just calling it `taxLabel`?